src/styles/_Checkout.scss

```diff
@@ -4,14 +4,14 @@
 .checkout {
   min-height: 100vh;
   padding: .5rem 2.5rem;
-  font: v.$ternary-font;
+  @include m.body-font;
```

Comment:
thanks for fixing that

Reply:
You're welcome! I'm glad you had the font variable set up already - made it very easy to change.